### PR TITLE
fix text in footer add attribute plain text

### PR DESCRIPTION
**Pull request by Iryna-Alchynska**

### CODE REVIEW

File: components/sections/Footer/index.js [hunk 1]
```diff
@@ -43,6 +43,9 @@ module.exports = cx.contentElement
         cx.part.plainText
             .withId('a9193fda-5f84-11ed-9b6a-0242ac120002')
             .withLabel('Copyright Text'),
+        cx.part.plainText
+            .withId('fcda9f56-f3b8-11ed-a05b-0242ac120003')
+            .withLabel('Agree Text'),
```
Comment (dreamy-dev) [inline components/sections/Footer/index.js:48]: Please rename to "Terms and Conditions Text".

File: components/sections/Footer/template.twig [hunk 2]
```diff
@@ -1,43 +1,125 @@
...
+                © 2022 JULIUS BAER
+            </p>
         </div>
     </div>
     <div class="footer_bottom">
         <div class="footer_disclaimer">
             <div class="footer_plainText">
-                I have read and agree to the
-                <a id="a9193e2c-5f84-11ed-9b6a-0242ac120002" href="[Terms_of_use]" data-bsi-element-part="link" class="footer_bottomLink">Terms of Use</a>
+                <p data-bsi-element-part="plain-text"
```
Comment (dreamy-dev) [inline components/sections/Footer/template.twig:81]: This doesn't work for several reasons:
- It breaks the layout:
image

- The Text is not fully editable, only the first part can be edited

File: components/sections/Footer/template.twig [hunk 3]
```diff
@@ -1,43 +1,125 @@
...
+                <a id="a9193be8-5f84-11ed-9b6a-0242ac120002"
+                    href="[Data_Privacy]"
+                    data-bsi-element-part="link"
+                    class="footer_bottomLink">
+                    Data Privacy Policy
+                </a>
             </div>
         </div>
         <div class="footer_backToTop">
-            <a id="a9193ad0-5f84-11ed-9b6a-0242ac120002" data-bsi-element-part="link" href="#top">Back to top</a>
```
Comment (dreamy-dev) [inline components/sections/Footer/template.twig:36]: Did you change anything here or are these your prettier settings that cause these changes?

**Review by dreamy-dev — CHANGES_REQUESTED** [review]

File: components/sections/Footer/template.twig (hunk 3 above)
Comment (Iryna-Alchynska) [reply components/sections/Footer/template.twig:36]: it is prettier settings

File: components/sections/Footer/index.js (hunk 1 above)
Comment (Iryna-Alchynska) [reply components/sections/Footer/index.js]: changes are done. Now this part of the footer is formatted element part

**Pull request merged**